docs/source/getting-started/content/proc_starting-the-virtual-machine.adoc

```diff
@@ -37,5 +37,6 @@ $ {bin} start --vm-driver virtualbox --bundle _path_to_system_bundle_
 
 [NOTE]
 ====
-The cluster takes a minimum of four minutes to start the necessary containers and operators before serving a request.
+* The cluster takes a minimum of four minutes to start the necessary containers and operators before serving a request.
+* If errors are encountered on the start command due to a mismatch in the hyperkit bundle, use the '{bin} delete' command to remove older versions.  These older versions may have user data and care should be taken before issuing the command to insure past work is not lost. 
```

Comment:
This is not specifc to Hyperkit, so this is incorrect.

Reply:
I modified the message to generalize for other dependencies and referenced the hyperkit bundle as an example.  I could make it more specific if there is a set list of dependencies, however I expect this could change over time.  Also I would expect that the support on start or setup handles this better in the future to the point where this not is no longer needed.  Rather than wait until that point, I wanted to get something else out for others to find to avoid spending time trying to resolve this.